lib/src/main/kotlin/starknet/provider/JsonRpcMethod.kt

```diff
@@ -0,0 +1,5 @@
+package starknet.provider
+
+enum class JsonRpcMethod(val value: String) {
```

Comment:
`val name: String`

Reply:
Changed to 'method', as 'name' is already used in enum class and would have to be overriden